data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,2723 @@
+			choice
+				`	"Let's go then!"`
+					goto leave
+				`	"Why can't you just use a jump drive and go there yourself?"`
+				`	"What will my cut be?"`
+					goto cut
+			`	"Jump drives are a very prized possession among my people. I would not risk one by travelling through hostile space. You had access to a jump drive, so you should know about the hostiles to the south. The aliens to the north would not be very happy to see me either... A jump drive would not be of any help to us." He seems disappointed by you having to ask such an obvious thing.`
+			choice
+				`	"Let's go then!"`
+					goto leave
+				`	"Sorry, but I won't risk my ship by going anywhere close to these pirates."`
+					decline
+			label cut
+			`	"You humans always seem obsessed with money... It is a large sum. I will only take one quarter of it, and you know how much my people value jump drives."`
+			label leave
+			`	"One last thing: We should not go there by jump drive, especially considering where we are going. It would only attract unwanted attention to us. Besides, we should not travel through Wanderer space; I do not want to try their patience or tolerance considering you have some association with us." You accompany him to your ship, and he admits to quite liking the interior, as well as the general look of it. Your ship "is adequate for a warrior," he says.`
+				accept
+	on visit
+		dialog "You do not have the jump drives. You either did not go to <destination> to get the jump drives yet, or you left your escort allocated to carrying the jump drives behind - better launch and wait for it."
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	to complete
+		has "Unfettered: Jump Drive Retrieval: offered"
+	on complete
+		outfit "Jump Drive" -4
+		payment 3000000
+		log `Helped a young Unfettered captain to bring jump drives from a pirate world.`
+		log "People" "Si-Ra" `A young, impetuous Unfettered captain who seems to grow wiser by the day. He has impressive leadership and combat skills, and seems to be a fan of acquiring bigger and bigger ships. He's always trying to improve his tactics.`
+		conversation
+			`"Here is your share: <payment>. You have saved my life, Captain <first>, and I shall never forget it. You have earned the right to know my name: "Si-Ra". It is one I deserve and wear proudly. Come see me again soon if you are up for a challenge. I'm sure that I will have what you want. In the meantime, make sure to check the job boards for more missions like this one."`
+			branch "hydra question"
+				not "Unfettered: Attracted Attention: offered"
+			choice
+				`	(Let him leave).`
+					goto leave
+				`	"I heard you say a name, 'Hydra', I believe."`
+			action
+				event "hydra revealed"
+			`	"If you really must know, that is how the pirates who attacked us are called." He obviously does not wish to elaborate on it any further.`
```

Comment:
```suggestion
			`	"If ye really must know, that be how the pirates who attacked us are called." He obviously does not wish to elaborate on it any further.`
```

Reply:
Maybe we could add a creative way to call those pirates? I dont think he'd think highly of them

Reply:
Done. I was looking for a place to wedge that in from my reference list ;)